src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,29 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
         try {
             user = authenticationService.login("bob", "1234");
+            logger.info("User {} logged in successfully.", getUserLogin(user));
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Login failed for user {} with message: {}",
```

Comment:
The logger.error() statement also uses the method getUserLogin(user) that could return an empty string. It would be more useful to include more specific user information in the error message, if available, to aid in debugging.